CRM/Member/Form.php

```diff
@@ -77,6 +77,23 @@ class CRM_Member_Form extends CRM_Contribute_Form_AbstractEditPayment {
   public $_priceSetId;
 
   /**
+   * Used when price set specified.  Used to indicate that form is only to retrieve
+   * price set.  How we use price set is unconventional.  We re-use the same
+   * Membership/MembershipRenewal Form used to enter memberships.  When user selects
+   * a PriceSet, we use JavaScript to inject HTML returned by a recursive call to this form.
+   *
+   * This flag controls this recursive call, short-circuiting it to only retrieve the price
```

Comment:
This is a pattern we have been aggressively removing elsewhere - we need to think about another way for this